packages/ui/panda/src/components/Card/Card.tsx

```diff
@@ -0,0 +1,109 @@
+import { cx, css } from '_panda/css';
+import { flex } from '_panda/patterns';
+
+// 아예 외부 경로로 옮겨도 괜찮을것 같다.
+const ANIMAL_CARD_IMAGE_BASE_URL = '/animal-card/';
+
+interface Props {
+  tier: 'EX' | 'S_PLUS' | 'A_PLUS' | 'B_MINUS';
+  type: string;
+  dropRate: string;
+  personaImage: string;
+}
+
+export function Card(props: Props) {
+  const { bg, thumbnail } = getTierToCardInfo(props.tier);
+
+  return (
+    <div className={cx('animal-card-container', container)}>
+      <div className={bgImage}>
+        <img src={ANIMAL_CARD_IMAGE_BASE_URL + bg} alt={props.tier} width={265} height={328} />
+      </div>
+      <div className={thumbnailImage}>
+        <img src={ANIMAL_CARD_IMAGE_BASE_URL + thumbnail} alt={props.tier} width={233} height={233} />
+      </div>
+      <picture className={thumbnailImage}>
+        <img src={props.personaImage} alt={props.type} width={233} height={233} />
+      </picture>
+      <div className={cx('animal-card-info', infoWrapper)}>
+        <p className={cx('animal-card-type', typeText)}>{getAnimalTypeLabel(props.type)}</p>
+        <p className={cx('animal-card-rating', ratingText)}>{props.dropRate}</p>
+      </div>
+    </div>
+  );
+}
+
+const getAnimalTypeLabel = (type: string) => {
+  const words = type.split('_');
+  return words.map((word) => word.charAt(0).toUpperCase() + word.slice(1).toLowerCase()).join(' ');
+};
```

Comment:
요 함수는 한 눈에 알아보기 어려운데 😢 

간단한 예제를 tsdoc으로 추가하면 좋을 거 같아요 
```suggestion
/**
* @description `foo` to `bar`
*/
const getAnimalTypeLabel = (type: string) => {
  const words = type.split('_');
  return words.map((word) => word.charAt(0).toUpperCase() + word.slice(1).toLowerCase()).join(' ');
};
```